src/shared/components/home/site-sidebar.tsx

```diff
@@ -54,7 +51,7 @@ export class SiteSidebar extends Component<SiteSidebarProps, SiteSidebarState> {
 
   siteName() {
     return (
-      <>
+      <div className={!this.state.collapsed ? "mb-2" : undefined}>
```

Comment:
If the 2nd condition is empty, just do `!this.state.collapsed && "mb-2"` , or ever better, use the `classNames` util.